src/appleseed/renderer/modeling/light/sunlight.cpp

```diff
@@ -85,7 +85,9 @@ namespace
 
     // Solid angle sustained by the Sun, as seen from Earth (in steradians).
     // Reference: http://en.wikipedia.org/wiki/Solid_angle#Sun_and_Moon
-    const float SunSolidAngle = 6.87e-5f;
+    float SunSolidAngle = 6.87e-5f;
```

Comment:
`SunSolidAngle` is now a global variable, which will cause all kinds of problems, not being thread-safe being the biggest one. You need to make it a class member (and rename it to `m_sun_solid_angle` as class members should be).